src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -524,4 +524,7 @@
   <data name="Convert_for_to_foreach" xml:space="preserve">
     <value>Convert 'for' to 'foreach'</value>
   </data>
+  <data name="Convert_foreach_to_for" xml:space="preserve">
+    <value>Convert 'foreach' to 'for'</value>
```

Comment:
@ivanbasov @sharwell where did we land with naming of these features.  i did "Convert 'for' to 'foreach'".  But Ivan, did "Convert to 'foreach'".   We should make this all consistent.

Reply:
@CyrusNajmabadi, @sharwell, we had a meeting yesterday. We confirmed that we keep names like:
- Convert to 'for'
- Convert to 'foreach'
- Convert to query
- Convert to 'fluent syntax'

Reply:
@ivanbasov I like those . Can you update my feature with 'convert to 'foreach'' as part of your PR?

Reply:
Sure. I will do. Thank you, @CyrusNajmabadi!